.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -9,14 +9,13 @@ body:
       description: What Apache Iceberg version are you using?
       multiple: false
       options:
-        - "0.14.1 (latest release)"
+        - "1.0.0 (latest release)"
+        - "0.14.1"
         - "0.14.0"
         - "0.13.1"
         - "0.13.0"
         - "0.12.1"
         - "0.12.0"
-        - "0.11.1"
```

Comment:
assumed that even if someone reports issue in this version, they have to try it in latest version as this version is very old. So, removed it.

I can revert it if someone feels it is needed.

Reply:
not strong on this, but I would probably just leave it in there

Reply:
Yea let's just revert it, not sure why target this one specifically.

Reply:
ok. I have reverted it now.